code/go/0chain.net/smartcontract/zcnsc/burn_test.go

```diff
@@ -36,7 +36,7 @@ func Test_FuzzyBurnTest(t *testing.T) {
 	contract := CreateZCNSmartContract()
 	ctx := MakeMockStateContext()
 
-	burn, err := contract.Burn(tr, payload.Encode(), ctx)
+	burn, err := contract.Burn(tr, payload.Encode(), ctx) //nolint: typecheck
```

Comment:
Same as in `authorizer_test.go`. If declaring ctx explicitly worked before then use it here as well.
```go
	var ctx cstate.StateContextI = MakeMockStateContext()
```